tests/nc-end-to-end.spec.ts

```diff
@@ -1,10 +1,15 @@
 import { test, expect } from '@playwright/test';
+import { runNcEndToEndTest, verifyCurrentUrl, saveResults } from './helpers';
+import { testUsers } from './helpers/utils/test-data';
```

Comment:
@mexi-cano It looks like these imports are not used. testUsers is particular looks like it could be useful here to fill in the form values, I would suggest we either use it or delete it.